app/pages/project/classify.jsx

```diff
@@ -88,33 +86,40 @@ export class ProjectClassifyPage extends React.Component {
   }
 
   componentDidUpdate(prevProps) {
-    const { actions, currentClassifications, project, upcomingSubjects, workflow } = this.props;
+    const { actions, classification, project, upcomingSubjects, workflow } = this.props;
```

Comment:
The logic in `coponentDidUpdate` could maybe do with some comments to explain what's going on?